src/makeRobotAccountant.js

```diff
@@ -7,6 +7,23 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return (firstNumber) => {
+    return (secondNumber) => {
+      count++;
+
+      if (count < 3) {
+        return firstNumber + secondNumber;
+      } else {
+        const result = count % 2 === 0
+          ? 'Bzzz... Error!'
```

Comment:
```suggestion
      if (count <= 3 || count % 2 !== 0) {
        return firstNumber + secondNumber;
      } else {
        return 'Bzzz... Error!'
```

Давай об'єднаємо ці дві перевірки і спростимо код